packages/modals/src/styled/StyledClose.ts

```diff
@@ -23,11 +28,10 @@ const colorStyles = (props: ThemeProps<DefaultTheme>) => {
       color: ${getColor(foregroundColor, 700, props.theme)};
     }
 
-    &[data-garden-focus-visible] {
-      box-shadow: ${props.theme.shadows.md(
-        getColor(backgroundColor, 600, props.theme, 0.35) as string
-      )};
-    }
+    ${focusStyles({
+      theme: props.theme,
+      hue: backgroundColor
+    })}
```

Comment:
Correct, the `focusStyles` utility actually adds:
```css
&:focus {
  outline: none;
}
```
along with the focus ring and HCM styling. So it is safe to replace it in place and preferable for the sake of LVHFA.
Another reason we willingly replace `data-garden-focus-visible` is because `focusStyles` uses it and results in the same code (along with adding the native CSS selector `:focus-visible`).